docs/faq.md

```diff
@@ -155,10 +155,12 @@ To work around exceptional cases, you can mount a docker config into
 the Flux container. See the argument `--docker-config` in [the daemon
 arguments reference](references/daemon.md).
 
-For ECR, if you are using Kiam, you need to whitelist the following API routes:
-```
---whitelist-route-regexp=(/latest/meta-data/placement/availability-zone|/latest/dynamic/instance-identity/document)
-```
+For ECR, flux requires access to the ec2 instance metadata api to obtain aws credentials. Kube2iam, Kiam, and potentially other kuberenetes iam utilities may block pod level access to the ec2 metadata apis. If this is the case, flux will be unable to poll ecr for automated workloads
```

Comment:
Small nitpick on capitalization of acronyms.

```suggestion
For ECR, Flux requires access to the EC2 instance metadata API to obtain AWS credentials. Kube2iam, Kiam, and potentially other Kubernetes IAM utilities may block pod level access to the EC2 metadata APIs. If this is the case, Flux will be unable to poll ECR for automated workloads
```

In addition to this, can you please split this to multiple lines, we try to maintain a width of `<=80` in our documentation docs.

Reply:
Capitalised and wrapped :capital_abcd: :gift: